docs/Developers/Newcomers_Guide.rst

```diff
@@ -146,7 +154,22 @@ Step 5. Sending Your Changes
 ----------------------------
 
 Now that you've fixed the issue, you've tested it and you think it is ready
-to be merged, create a commit and push it to your fork, using:
+to be committed.
```

Comment:
Hmm, I might just be too picky here, but this sentence seems to dangle. Perhaps you could terminate it (by adding "you should check your branch") or by moving the note earlier or later.